src/main/fc/fc_tasks.c

```diff
@@ -445,6 +446,15 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
+#ifdef USE_LIGHTS
+    [TASK_LIGHTS] = {
+        .taskName = "LIGHTS",
+        .taskFunc = lightsUpdate,
+        .desiredPeriod = TASK_PERIOD_HZ(100),     // 100 Hz
```

Comment:
Isn't 100Hz an overkill? Do we want to update light status 100 times per second? I think 10Hz whould be enough.

Reply:
It is needed for the failsafe lights flashing code. As the default I set pulses the lights 100ms every second. It is what I use to keep battery usage low in case the model is lost and a long time is needed to find it. If we set it to 10Hz it won't be possible as it will make the lights either not light up or light up 100ms or 200ms each cycle depending on the time variability between task runs. The task priority is set to LOW it shouldn't be a problem.